docs/02-hugo.Rmd

```diff
@@ -1,28 +1,83 @@
 
-Why Hugo?
-
-- fast
-- easy to install (single binary)
-- a big community (Github)
+Hugo is one example of a static site generator. It provides the framework for a variety of website types including, but certainly not limited to, blogs. We are fans of Hugo for many reasons but there are a few that stand out. Unlike other static site generators, Hugo doesn't require any dependencies, making it very quick and easy to install on most operating systems. It was also designed to render hundreds of pages of content faster than comparable static site generators and can reportedly render a single page in approximately 1 millisecond. Lastly, the community of Hugo users is very active both on the Hugo discussion boards^[https://discuss.gohugo.io/] and on GitHub^[https://github.com/spf13/hugo/issues].
 
 Although we think Hugo is a fantastic static site generator, there is really one and only one important but missing feature: the support for R Markdown. That is basically the whole point of the **blogdown** package. This missing feature means that you cannot easily generate results using R code on your web pages, since you can only use static Markdown documents. Besides, you have to use Hugo's choice of the Markdown engine named "Blackfriday" instead of the more powerful Pandoc.^[There is a feature request in Hugo's repository that has been three years old: https://github.com/spf13/hugo/issues/234, and it seems that it will not be implemented in the near future.]
 
-What does a typical file and directory structure look like?
 
-- config.toml
-- content
-- static
-- themes
-- layouts
+Hugo uses a consistent file and folder structure to create your website.  The rest of this chapter will give more detail on the following files and folders:
+
+- `config.toml`
+- `content/`
+- `static/`
+- `themes/`
+- `layouts/`
+
+```{r folders, fig.cap='Files and Folders created when you create a new site using **blogdown**', echo=FALSE, fig.align='center', out.width='90%'}
+knitr::include_graphics('images/folder-structure.png')
+```
 
 ## Configuration
 
-TOML syntax
+The first file that Hugo uses to generate a website is the configuration or `config` file. It is worth mentioning here that technically speaking, Hugo could generate your website without a `config` file if your files and folders are in the correct locations. Regardless, Hugo will look for a `config` file in the root directory of your website, so it is best to make sure that you have one. Generally speaking, this is the only file that Hugo expects to be in the root directory. 
+
+As its name suggests, the `config` file is where you tell Hugo exactly how to render your webpage and display your menus. It is also the location of other global parameters like links to your social networks, your site's name, and the base URL for your website. 
```

Comment:
I guess it is not easy to guess that the config file `config.toml` is used to "display your menus", so perhaps readers cannot follow "As its name suggests". I'd just say something like "config.toml is for setting global options for your site", and give a few examples of these options (which you have done).